docs/spec.md

```diff
@@ -614,13 +630,7 @@ Performs element-wise addition of two tensors `lhs` and `rhs` and produces a
 
 #### Constraints
 
-* (C1) `shape(lhs) = shape(rhs) = shape(result)`.
-* If the operation uses non-quantized tensors:
-  * (C2) `element_type(lhs) = element_type(rhs) = element_type(result)`.
-* If the operation uses quantized tensors:
-  * (C3) `element_type(lhs) = element_type(rhs) = element_type(result)`,
-    except for quantization parameters which may differ.
-  * (C4) `quantization_dimension(lhs)` is empty.
```

Comment:
[No action]

Just to note: we will capture `C4` as part of https://github.com/openxla/stablehlo/pull/1566 and https://github.com/openxla/stablehlo/issues/1574.